src/helpers/createFounderProof.ts

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios'
+import Signature from 'models/Signature'
+import env from './env'
+import unpackSignature from './unpackSignature'
+import { getYCAllowMapInput } from './getYCMerkleTreeProof'
+import PublicKey from 'models/PublicKey'
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+declare const snarkjs: any
+
+const baseURL = `${env.VITE_VERIFY_URL}/v0.2.1`
+
+export async function requestTwitterSignature(token: string) {
+  const { data } = await axios.post<Signature>(`${baseURL}/verify-yc/twitter`, {
+    token,
+  })
+  return data
+}
+
+export async function getEddsaPublicKey() {
+  const { data } = await axios.get<PublicKey>(
+    `${baseURL}/verify/eddsa-public-key`
+  )
+  return data
+}
+
+export default async function (token: string) {
+  try {
+    const response = await fetch('/trees/yc/1.json')
```

Comment:
It fetches it's own file? Can we simply do `require` here?

Reply:
Or maybe we can import this file?

Reply:
In theory, you can only fetch from the public folder